pkgs/development/tools/spring-boot/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchzip, jdk, makeWrapper, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "spring-boot";
+  version = "2.1.8";
+
+  src = fetchzip {
+    url = "https://repo.spring.io/release/org/springframework/boot/${pname}-cli/${version}.RELEASE/${pname}-cli-${version}.RELEASE-bin.zip";
+    sha256 = "0shkznvv8abb365pgzfp265ihmf17mqigzzimdkzyp9jrh9kgiqx";
+  };
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  installPhase = ''
+    rm bin/spring.bat
+    installShellCompletion --bash shell-completion/bash/spring
+    installShellCompletion --zsh shell-completion/zsh/_spring
+    rm -r shell-completion
+    cp -r . $out
+    wrapProgram $out/bin/spring \
+      --prefix JAVA_HOME : ${jdk}
+  '';
+
+  meta = with stdenv.lib; {
+    description = ''
```

Comment:
The description might be shorter. From [the manual](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes):

> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.

Reply:
Thanks for pointing out the manual.